tools/cli/helpers/util.js

```diff
@@ -169,6 +169,63 @@ function runPrettier(pathToFile) {
   }
 }
 
+/**
+ * Move to directory
```

Comment:
Update comments according to conventions (https://github.com/airbnb/javascript#comments)

Reply:
Done